style.css

```diff
@@ -1,1242 +1,1104 @@
-/* stylelint-disable no-missing-end-of-source-newline */
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
 
-* {
-  padding: 0;
-  margin: 0;
+html {
   box-sizing: border-box;
+  font-size: 62.5%;
 }
 
-a {
-  text-decoration: none;
+body {
+  overflow-x: hidden;

```

Comment:
- [ ]  Please, *As suggested by the previous reviewer,* there is still, a horizontal scroll on the mobile view which means your project is not responsive on the mobile view so make sure that you remove the horizontal overflow on the mobile view. 😎

_a general suggestion, the cause of horizontal scrolling on mobile devices is often due to elements on the page being too wide for the viewport._

https://www.loom.com/share/dc9fbbdcfc8448619d0af50997927ddf